### PR TITLE
make bsb, grib and mrf follow disabling standard of #1250

### PR DESCRIPTION
**Pull request by tbonfort**
Rename ./configure arguments --without-bsb, --without-grib and --without-mrf to --disable-driver-bsb --disable-driver-grib and --disable-driver-mrf to be inline with #1250

### CODE REVIEW

**Comment by rouault** [comment]
The mrf issue is due to the fact that, when JPEG is enabled, we have to run "make" in frmts/jpeg to generate the frmts/jpeg/libjpeg12 files, before make can be run in frmts/mrf (on a only generated tree, you need to run 'make clean' in frmts/jpeg to reproduce).
Looking at frmts/nitf/GNUmakefile that has the same dependency to libjpeg12, there's an explicit rule in it to build libjpeg12. frmts/mrf/GNUmakefile likely needs the same

Can you squash the commits once done ?

**Comment by tbonfort** [comment]
@rouault this PR is rebased and ready for merge as soon as the CI finishes

File: gdal/configure.ac [hunk 1]
```diff
@@ -1463,7 +1463,23 @@ if test "$LIBZ_SETTING" != "no" ; then
   fi 
 fi
 
+dnl ---------------------------------------------------------------------------
+dnl LERC and MRF are coupled
```
Comment (rouault) [inline gdal/configure.ac]: Hum, are they ?
Since GDAL 2.4.0, the GTiff driver can build a LERC TIFF plugin, and this is not related at all with MRF
And normally (didn't try recently though), the MRF driver should be able to build without LERC

Comment (tbonfort) [reply gdal/configure.ac]: hmm, my bad, I was under the idea that lerc required mrf, but looking at the code that is clearly not the case, I'll revert the lerc part of this PR. FYI, mrf builds fine without lerc, and that was accounted for in the proposed change

**Comment by tbonfort** [comment]
@rouault : ready for merge
